examples/ios/swift/Migration/Examples/Example_v0.swift

```diff
@@ -0,0 +1,51 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+// swiftlint:disable orphaned_doc_comment
+// swiftlint:disable comment_spacing
+// swiftlint:disable mark
+
+import Foundation
+import RealmSwift
+
+//// MARK: - Schema
```

Comment:
Why is all of this commented out?

Reply:
What if the Realm API changes? we won't be to catch any compiler errors

Reply:
This will be fixed as soon as we have decided on config vs. target.